backend/tournaments/tools.py

```diff
@@ -72,35 +72,66 @@ def get_tournament_buckets(tournament):
 
   Return:
     List of dictionaries [{'name': <tournament name>,
-                           'members': [<username>, ...]}, ...]
+                           'members': [{
+                              'name': <player name>,
+                              'flag': <player country code>,
+                              'rating': <fixed rating>
+                              }, ...]}, ...]
 
   Throws:
     NotFound -- if tournament is not found.
   """
+  ratings = {}
+  for player in get_tournament(tournament).players.all():
+    ratings[player.member.name()] = player.fixed_rating
   buckets = get_tournament(tournament).bucket_set.all()
   return [
       { 'name': b.name,
-        'members': [ m.member.name() for m in b.players.all() ] }
+        'members': [ 
+      { 'name': m.member.name(),
+        'flag': m.member.country,
+        'rating': ratings[m.member.name()] }
```

Comment:
What will populate fixed ratings into database? I guess it will be the bot as nothing else has access to player's rating.
If so, is it possible that user doesn't have rating assigned yet?